src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/PictureBoxActionListTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.ComponentModel.Design;
+
+namespace System.Windows.Forms.Design.Tests
+{
```

Comment:
Let's make this a file scoped namespace. The end curly brace will need to be removed as well.
```suggestion
namespace System.Windows.Forms.Design.Tests;
```